spec/listeners/automation_rule_listener_spec.rb

```diff
@@ -146,6 +152,117 @@
     end
   end
 
+  describe '#conversation_updated with contacts attributes' do
+    before do
+      Contact.first.update!(custom_attributes: { customer_type: 'platinum', signed_in_at: '2022-01-19' },
```

Comment:
It's better to update the specific `contact` that we target in these test cases, rather than multiple ones.. For someone reading the tests, they would have to guess which one would be `Contact.first`.

I think for the following test cases, you are looking at just the `contact` or `conversation.contact` ?

Reply:
`Conversation.contact`